app/components/cards/player-card/page.tsx

```diff
@@ -32,25 +32,33 @@ const PlayerCard = (props: PlayerCardProps) => {
 			alignItems="center"
 			sx={{
 				borderBottom: 1,
-				borderColor: "#f5f5f4",
+				borderColor: "#E5E5E5",
 				display: "inline-flex",
 				mt: 1,
-				height: 100,
+				height: 60,
+				width: 480,
 			}}
 		>
-			<Grid 
-				item 
-				container 
-				xs={4}
-				alignItems="center"
-			>
-				<Avatar></Avatar>
-				<Typography sx={{ fontWeight: 700, ml: 2 }}>{props.name}</Typography>
+			<Grid item container xs={6} alignItems="center">
+				<Box
+					sx={{
+						ml: 2,
+						backgroundImage:
+							props.image ||
+							"linear-gradient(207deg, #EAE68E 13.76%, #FBBEBE 60.61%, #BEE1FB 100%);",
+						borderRadius: "50%",
+						width: 40,
+						height: 41,
+					}}
+				></Box>
+				<Typography sx={{ fontWeight: 700, ml: 2 }}>
+					{props.name}
```

Comment:
no props.something destructure it in the params

 PlayerCard: React.FC<interface> = ({name}) => {}